docs/dev/plugins/cloudformation/mapping.md

```diff
@@ -0,0 +1,24 @@
+# AWS CloudFormation
+
+## Mapping
+
+Mapping (none-) normative Types of TOSCA to AWS Resources.
+
+| (none-) normative Type | AWS Resource | Comment |
+| --- | --- | --- |
+| Compute | [AWS EC2](https://aws.amazon.com/de/ec2/) | A AWS-SecurityGroup has to be created |
+| SoftwareComponent | --- | Add scripts to the EC2 that is launched |
```

Comment:
We are currently discussing in #211 that modeled node templates might come without scripts.
As far as i can tell we might need some kind of internal (plugin-independent) script repository which supplies the plugins with needed scripts.
However, maybe a more elegant way would be to use beanstalk instead of ec2.

Reply:
Yes this would be internal scripts if not specified different. I add them to the template so e.g. apache will be installed.
As far as I know beanstealk wouldn't work here because we just have an SoftwareComponent. I added Beanstalk as possibility to web application.